docs/trusted-resources.md

```diff
@@ -17,7 +17,7 @@ Verification failure will mark corresponding taskrun/pipelinerun as Failed statu
 ## Instructions
 
 ### Sign Resources
-For `Sign` cli you may refer to [experimental repo](https://github.com/tektoncd/experimental/tree/main/pipeline/trusted-resources) to sign the resources. We're working to add `sign` and `verify` into [Tekton Cli](https://github.com/tektoncd/cli) as a subcommand.
+We have added `sign` and `verify` into [Tekton Cli](https://github.com/tektoncd/cli) as a subcommand in release [v0.28.0](https://github.com/tektoncd/cli/releases/tag/v0.28.0). Please refer to [cli docs](https://github.com/tektoncd/cli/blob/main/docs/cmd/tkn_task_sign.md) to sign and Tekton resources.
```

Comment:
Suggest:

Tekton CLI (v0.28.0 and later) support `sign` and `verify` subcommands.

Reply:
thanks!